src/index.js

```diff
@@ -120,7 +120,9 @@ function getMatchRanking(testString, stringToRank, options) {
     return rankings.ACRONYM
   }
 
-  return stringsByCharOrder(testString, stringToRank)
+  // will return a number between 0 and 1 depending
```

Comment:
Let's use the ranking rather than the actual number in this comment, just in case the ranking change in the future. Also, the next comment needs a space